server/dash/dashboards/neighborhood.py

```diff
@@ -43,35 +47,67 @@ def populate_options():
 # Define callback to update graph
 @app.callback(
     Output('graph1', 'figure'),
-    [Input("council_list", "value")]
+    [Input("council_list", "value"),
+    Input("data_type", "value")]
 )
-def update_figure(selected_council):
-
+def update_figure(selected_council, selected_timeframe):
     neighborhood_sum_df = df[df.council_name == selected_council].groupby(['created_date']).agg('sum').reset_index()  # noqa
     total_sum_df = df.groupby(['created_date']).agg('sum').reset_index()
-    total_sum_df["nc_avg"] = total_sum_df["counts"] / 99
-    merged_df = neighborhood_sum_df.merge(total_sum_df["nc_avg"].to_frame(), left_index=True, right_index=True)  # noqa
 
+    total_sum_df['nc_ma'] = total_sum_df.counts.rolling(selected_timeframe, center=True).mean()/NEIGHBORHOOD_COUNCILS_NUMBER
```

Comment:
Add spaces around `/`

Reply:
formatter moved it to next line, is that fine?